packages/esm-patient-medications-app/src/order-basket/order-basket.component.tsx

```diff
@@ -83,9 +83,16 @@ const OrderBasket = connect<OrderBasketProps, OrderBasketStoreActions, OrderBask
       orderDrugs(items, patientUuid, abortController).then((erroredItems) => {
         setItems(erroredItems);
         fetchActivePatientOrders();
-
-        if (erroredItems.length == 0) {
+        if (erroredItems.length !== 0) {
```

Comment:
Beforehand the workspace was closed if no items errored out, now its closed if something got an error. Does not really sound right to me.

Reply:
Made another pr.Check  MF-600:Added a confirm message #331